cinnabar/plotting.py

```diff
@@ -97,6 +98,8 @@ def _master_plot(
     statistic_type : str, default 'mle'
         the type of statistic to use, either 'mle' (i.e. sample statistic)
         or 'mean' (i.e. bootstrapped mean statistic)
+    scatter_kwargs : dict, default {}
```

Comment:
Could you update the default value here too please?

Reply:
Yes, sorry about that. Done